README.md

```diff
@@ -77,6 +77,53 @@ DESTDIR="$(pwd)/Lite XL.app" meson install --skip-subprojects -C build
 Please note that the package is relocatable to any prefix and the option prefix
 affects only the place where the application is actually installed.
 
+## Installing Prebuilt
+
+Head over to [releases](https://github.com/lite-xl/lite-xl/releases) and download the version for your operating system.
+
+### Ubuntu
```

Comment:
I think that the instructions are generic enough that this can be replaced with `Linux`.